pkg/get/tools.go

```diff
@@ -3487,5 +3487,32 @@ https://github.com/{{.Owner}}/{{.Repo}}/releases/download/{{.Version}}/{{.Name}}
 				{{- end -}}
 				https://github.com/{{.Owner}}/{{.Repo}}/releases/download/{{.Version}}/{{.Name}}-cli_{{.Version}}_{{$os}}_{{$arch}}.{{$extStr}}`,
 		})
+
+	tools = append(tools,
+		Tool{
+			Owner:       "replicatedhq",
+			Repo:        "replicated",
+			Name:        "replicated",
+			Description: "CLI for interacting with the Replicated Vendor API",
+			URLTemplate: `
+				{{$arch := "386"}}
```

Comment:
FYI we don't support 386 in any other apps, could you remove support?

Reply:
We support in the case of `helmfile`: https://github.com/alexellis/arkade/blob/d8986d8582d4706e986c3e7695cdc69abc15120b/pkg/get/tools.go#L88

Reply:
Removed it.

But replicated-cli has a distribution for arch `386`: https://github.com/replicatedhq/replicated/releases/download/v0.45.0/replicated_0.45.0_linux_386.tar.gz

and in helmfile we add arch `386`: https://github.com/alexellis/arkade/blob/master/pkg/get/tools.go#L88